app/src/models/allergy.model.ts

```diff
@@ -0,0 +1,16 @@
+export enum Allergy {
```

Comment:
Foretrekker `as const` for bedre typesupport: https://www.youtube.com/watch?v=jjMbPt_H3RQ

```
export const Allergy = {
    GLUTEN: "Gluten"
    ..
} as const;
```